content/webapp/components/PhysicalItems/PhysicalItems.tsx

```diff
@@ -41,52 +41,21 @@ const getItemsState = (items: PhysicalItem[]): ItemsState => {
     : 'up-to-date';
 };
 
-const useItemsState = (
-  items: PhysicalItem[]
-): [ItemsState, (s: ItemsState) => void] => {
-  /* https://github.com/wellcomecollection/wellcomecollection.org/issues/7120#issuecomment-938035546
-   *
-   * What if we don’t call the items API? There are two possible error cases:
-   *
-   * (1) We don’t show a “request item” button when an item is ready to request.
-   *     This could occur if an item was on hold, has been returned to the stores,
-   *     and we haven’t had the update through the catalogue pipeline yet.
-   * (2) We show a “request item” button when an item isn’t available to request.
-   *     Run the same scenario in reverse: somebody has put the item on hold,
-   *     but we haven’t realised yet in the catalogue API.
-   *
-   * Error (2) is worse than error (1).
-   *
-   * To avoid them:
-   *
-   * Query the items API if the status is “temporarily unavailable”
-   * Query the items API if you want to display a button based on the catalogue API data
-   *
-   * In all other cases the items API would be a no-op.
-   */
-  const [itemsState, setItemsState] = useState<ItemsState>(
-    getItemsState(items)
-  );
-
-  useEffect(() => {
-    setItemsState(getItemsState(items));
-  }, [items]);
-
-  return [itemsState, setItemsState];
-};
-
 const PhysicalItems: FunctionComponent<Props> = ({
   work,
   items: workItems,
 }: Props) => {
   const { state: userState } = useUser();
   const [userHolds, setUserHolds] = useState<Set<string>>();
   const [physicalItems, setPhysicalItems] = useState(workItems);
-  const [itemsState, setItemsState] = useItemsState(workItems);
+
+  // https://github.com/wellcomecollection/wellcomecollection.org/issues/7120#issuecomment-938035546
+  const [itemsState, setItemsState] = useState(getItemsState(workItems));
 
   useEffect(() => {
+    setItemsState(getItemsState(workItems));
     setPhysicalItems(workItems);
-  }, [workItems]);
+  }, [work]);
```

Comment:
items will only change if the work changes, so using that as a dependency instead